pkg/statser/flush_notifier.go

```diff
@@ -0,0 +1,48 @@
+package statser
+
+import (
+	"sync"
+	"time"
+)
+
+type flushNotifier struct {
+	lock         sync.RWMutex
+	flushTargets []chan<- time.Duration
+}
+
+// RegisterFlush registers a channel which will receive a notification after every flush. If
+// the channel blocks, the notification will be silently dropped.  Thread-safe.
+func (fn *flushNotifier) RegisterFlush() (<-chan time.Duration, func()) {
```

Comment:
doc string doesn't mention the second returned value. Having it named would help too.